module-apps/apps-common/ApplicationCommon.cpp

```diff
@@ -468,12 +468,13 @@ namespace app
     sys::MessagePointer ApplicationCommon::handleMinuteUpdated(sys::Message *msgl)
     {
         if (state == State::ACTIVE_FORGROUND && getCurrentWindow()->updateTime()) {
-
             if (isOnPhoneLockWindow()) {
                 updateStatusBarOnPhoneLockWindow();
+                refreshTimeOnPhoneLockWindow();
```

Comment:
why not:
- override handleMinuteUpdated in the PhoneLockWindow
- change return type there
It would remove Window specific code fully from the ApplicationComomn